scripts/run_integration_tests.sh

```diff
@@ -57,20 +67,9 @@ for ((i = 0 ; i < ${#SERVERLESS_CONFIGS[@]} ; i++)); do
         exit 1
     fi
     echo "===================================="
-
 done
 
 if [ "$UPDATE_SNAPSHOTS" = "true" ]; then
-        cd $root_dir
-        BRANCH=$(git rev-parse --abbrev-ref HEAD)
-        if [ $BRANCH = "master" ]; then
-            echo "Error: Cannot update snapshot files directly through the master branch, please create a development branch then run the script again."
-            exit 1            
-        else
-            echo "Commiting and pushing up snapshot changes to ${BRANCH}. Please create a pull request on GitHub."
-            git add .
-            git commit -m "Update snapshots for integration tests"
-            git push origin $BRANCH
-        fi
+    echo "Commit the changes and create a PR"
```

Comment:
I don't think you necessarily need this message here -- what do you think?

Reply:
No its probably unnecessary